File: JavaScript/Basics/0.3-Conditional-Steatements-Advanced/Lab/personal-titles.js
function personTitles(input) {
    let age = Number(input[0]);
    let gender = input[1].toLowerCase();
    if (age < 16) {
        if (gender == "f") {
            console.log("Miss");
        } else if (gender == "m") {
            console.log("Master");
        }
    } else {
       if (gender == "f") {
            console.log("Ms.");
        } else if (gender == "m") {
            console.log("Mr.");
        }
    }
}

File: Python/0.1-Basics/0.3-Conditional-Statements-Advanced/Lab/personal_titles.py
age = float(input())
gender = input().lower()

if gender == "m":
    if age < 16:
        print(f"Master")
    else:
        print(f"Mr.")
else:
    if age < 16:
        print(f"Miss")
    else:
        print(f"Ms.")
